### PR TITLE
Internationalization of ingest messages

### LINKED ISSUE #9570: Untranslated error ingest message
_recherche.data.gouv team_

**What steps does it take to reproduce the issue?**
Getting file ingestion in error

* When does this issue occur?
When displaying the file after ingestion

* Which page(s) does it occurs on?
The dataset page

* What happens?
The message that the file is badly ingested is not translatable

* To whom does it occur (all users, curators, superusers)?
All the users

* What did you expect to happen?
Be able to configure the life labels of the properties files

https://github.com/IQSS/dataverse/blob/1a797171cdb73741b5da4a683f38697558349b5c/src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java#L131

https://github.com/IQSS/dataverse/blob/1a797171cdb73741b5da4a683f38697558349b5c/src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java#L133

**Which version of Dataverse are you using?**
5.12.1

**Any related open or closed issues to this bug report?**
https://github.com/IQSS/dataverse/issues/9488
https://github.com/IQSS/dataverse/issues/9427

capture

### PR DESCRIPTION
**Pull request by stevenferey**
**What this PR does / why we need it**:
Internationalization of ingest messages

**Which issue(s) this PR closes**:

- Closes #9570

### CODE REVIEW

File: src/main/java/propertyFiles/Bundle.properties [hunk 1]
```diff
@@ -1809,6 +1809,8 @@ file.spss-savEncoding.current=Current Selection:
 file.spss-porExtraLabels=Variable Labels
 file.spss-porExtraLabels.title=Upload an additional text file with extra variable labels.
 file.spss-porExtraLabels.selectToAddBtn=Select File to Add
+file.ingest.saveFailed.message=Ingest succeeded, but failed to save the ingested tabular data in the database; no further information is available
+file.ingest.saveFailed.detail.message=Ingest succeeded, but failed to save the ingested tabular data in the database:
```
Comment (pdurbin) [inline src/main/java/propertyFiles/Bundle.properties:1813]: Oh good, there's a space after the : even though it's hard to see.

**Review by pdurbin — APPROVED** [review]
This looks good to me. Thanks, @stevenferey!

I'm going to give this our smallest size estimate because it shouldn't be hard to test.

**Comment by pdurbin** [comment]
@stevenferey is a first time contributor (❤️ thanks! ❤️ ) so I had to click the "run" button for the checks.

**Comment by stevenferey** [comment]
Thanks for your feedback @pdurbin !

The error during the maven build is not to be related to the PR for me, is it?

Thanks,
Steven.

**Comment by pdurbin** [comment]
@stevenferey no, this error is unrelated to your pull request:

`Failed to read schema document 'https://ddialliance.org/Specification/DDI-Codebook/2.5/XMLSchema/codebook.xsd', because 1) could not find the document; 2) the document could not be read; 3) the root element of the document is not <xsd:schema>.`

Internally, we've been discussing how to fix this: https://iqss.slack.com/archives/C010LA04BCG/p1682360103211409

**Comment by kcondon** [comment]
@stevenferey Please refresh this branch from develop since there is an outdated flyway db script name in this branch that is causing trouble deploying. Thanks!

**Comment by pdurbin** [comment]
I took care of the merge.

**Comment by kcondon** [comment]
[like]  Condon, Kevin M reacted to your message:
________________________________

File: src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java [hunk 2]
```diff
@@ -128,9 +128,9 @@ public void onMessage(Message message) {
                             IngestReport errorReport = new IngestReport();
                             errorReport.setFailure();
                             if (ex.getMessage() != null) {
-                                errorReport.setReport("Ingest succeeded, but failed to save the ingested tabular data in the database: " + ex.getMessage());
+                                errorReport.setReport(BundleUtil.getStringFromBundle("file.ingest.saveFailed.detail.message") + ex.getMessage());
```
Comment (pdurbin) [inline src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java:131]: To artificially test this I used the following code...

```
     public boolean ingestAsTabular(Long datafile_id) {
+        if (true) {
+            throw (new RuntimeException("customMessage"));
+        }
         DataFile dataFile = fileService.find(datafile_id);
```

... to create this screenshot, which looks good to me:

Screen Shot 2023-05-26 at 2 47 48 PM

File: src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java [hunk 3]
```diff
@@ -128,9 +128,9 @@ public void onMessage(Message message) {
...
                             } else {
-                                errorReport.setReport("Ingest succeeded, but failed to save the ingested tabular data in the database; no further information is available");
+                                errorReport.setReport(BundleUtil.getStringFromBundle("file.ingest.saveFailed.message"));
```
Comment (pdurbin) [inline src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java:133]: For this one I used...

```
     public boolean ingestAsTabular(Long datafile_id) {
+        if (true) {
+            throw (new RuntimeException());
+        }
         DataFile dataFile = fileService.find(datafile_id);
```

... to get this screenshot, which also looks fine to me:

Screen Shot 2023-05-26 at 2 54 47 PM

**Review by pdurbin — APPROVED** [review]
I tested this by artificially adding some runtime exceptions, which I didn't commit, of course. Looks good to me. Thanks, @stevenferey!

**Pull request merged**